app/controllers/presentations_controller.rb

```diff
@@ -6,4 +6,27 @@ def show
   def new
     @presentation = Presentation.new
   end
+
+  def create
+    @presentation = Presentation.new(presentation_params)
+
+    if @presentation.save
+      flash[:notice] = 'Presentation was successfully created'
+      redirect_to @presentation
+    else
+      render 'new'
+    end
+  end
+
+  private
+
+  def presentation_params
+    params.require(:presentation)
```

Comment:
I would love to normalize our indentation a little bit. Unfortunately we're not quite sure how to enforce this with Rubocop yet but we don't to write this as:

```ruby
def presentation_params
   params.require(:presentation).permit(
    :presenter,
    :topic,
    :duration,
    :description,
    :person_id,
    :event_id
  )
 end
```